data/strings/strings.txt

```diff
@@ -21921,7 +21924,7 @@
 
   [dialog_routing_change_end_carplay]
     tags = ios
-    en = Unable to build a route to your destination. Please choose another point
+    en = Unable to build a route to your destination. Please choose another end point
```

Comment:
find a route, not build.

I don't see why this is different from the can't-find-route dialog above.  Is it the same issue, or is something different?

If someone wants to go to X, telling them to choose Y as a destination instead is goofy.

Reply:
Changed to find. The one above is mobile and this string is for carplay, where it may not be possible to change the start point.